nlmod/dims/resample.py

```diff
@@ -573,6 +518,8 @@ def structured_da_to_ds(da, ds, method="average", nodata=np.nan):
             raise ValueError(
                 "No extent or delr and delc in ds. Cannot determine affine."
             )
+        from .grid import get_affine
```

Comment:
why not a top level import? It is now a separate import for structured and vertex.

Reply:
Because the deprecated version of get_affine is also in resample.py. We do not want the get_affine from resample, but from grid. Not very pretty, but for now it works, and the user gets to see the deprecation message when he uses nlmod.resample.get_affine.